rust_icu_uloc/src/lib.rs

```diff
@@ -228,4 +312,55 @@ mod tests {
         let expected = ULoc::try_from("sr").expect("get sr locale");
         assert_eq!(minimized_subtags.label(), expected.label());
     }
+
+    #[test]
+    #[ignore = "ICU-20931"]
+    fn test_accept_language_fallback() {
+        let accept_list: Result<Vec<_>, _> = vec!["es_ES", "ar_EG", "fr_FR"]
+            .into_iter()
+            .map(|s| ULoc::try_from(s))
+            .collect();
+        let accept_list = accept_list.expect("make accept_list");
+
+        let available_locales: Result<Vec<_>, _> = vec!["de_DE", "en_US", "es_MX", "nl_NL"]
+            .into_iter()
+            .map(|s| ULoc::try_from(s))
+            .collect();
+        let available_locales = available_locales.expect("make available_locales");
+
+        let actual =
+            ULoc::accept_language(accept_list, available_locales).expect("call accept_language");
+        assert_eq!(
+            actual,
+            (
+                ULoc::try_from("es_MX").ok(),
+                UAcceptResult::ULOC_ACCEPT_FALLBACK
+            )
+        );
+    }
+
+    #[test]
+    fn test_accept_language_exact_match() {
+        let accept_list: Result<Vec<_>, _> = vec!["es_ES", "ar_EG", "fr_FR"]
+            .into_iter()
+            .map(|s| ULoc::try_from(s))
+            .collect();
+        let accept_list = accept_list.expect("make accept_list");
+
+        let available_locales: Result<Vec<_>, _> = vec!["de_DE", "en_US", "es_MX", "ar_EG"]
+            .into_iter()
+            .map(|s| ULoc::try_from(s))
+            .collect();
+        let available_locales = available_locales.expect("make available_locales");
+
+        let actual =
+            ULoc::accept_language(accept_list, available_locales).expect("call accept_language");
+        assert_eq!(
+            actual,
+            (
+                ULoc::try_from("ar_EG").ok(),
```

Comment:
feel free to add your original example with two-letter locales, except you add one additional unrelated locale at end?

Reply:
Done